storage/innobase/handler/ha_innodb.cc

```diff
@@ -7886,7 +7916,7 @@ ha_innobase::build_template(
 			bool index_contains = index->contains_col_or_prefix(
 				is_v ? num_v : i - num_v, is_v);
```

Comment:
Can we write just `i - num_v`?

Reply:
Replied to the other comment. We cannot ignore virtual columns at this point.